code/clue.js

```diff
@@ -7,53 +7,281 @@ const mrGreen = {
+const livingRoom = {
+  name: 'Living room',
+  image: 'assets/livingroom.jpg',
+}
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.png',
+}
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.jpg',
+}
+
+const guestHouse = {
+  name: 'Guest house',
+  image: 'assets/guesthouse.png',
+}
+
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.jpg',
+}
+
+// ARRAYS
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  prPlum,
+  mrsScarlet,
+  colMustard,
+  mrsPeacock,
+  mrsWhite,
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol,
+]
 
-const rooms = []
+const rooms = [
+  diningRoom,
+  conservatory,
+  kitchen,
+  study,
+  library,
+  billiardRoom,
+  lounge,
+  ballroom,
+  hall,
+  spa,
+  livingRoom,
+  observatory,
+  theater,
+  guestHouse,
+  patio,
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// OBJECT THAT KEEPS THE MYSTERY.
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: '',
+}
 
-// This function will be invoked when you click on the killer card.
+// Random killer function
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
 
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
+  document.getElementById('killerImage').src = mystery.killer.image
   document.getElementById(
     'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName} ${mystery.killer.age} ${mystery.killer.occupation}`
 }
+document.getElementById('killerCard').onclick = pickKiller;
+
+// Random weapon function
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponImage').src = mystery.weapon.image
+  document.getElementById(
+    'weaponName'
+  ).innerHTML = `${mystery.weapon.name} ${mystery.weapon.weight} lbs`
+}
+document.getElementById('weaponCard').onclick = pickWeapon;
+
+// Random room function
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms)
+  document.getElementById('roomImage').src = mystery.room.image
+  document.getElementById(
+    'roomName'
+  ).innerHTML = `${mystery.room.name}`
+}
+document.getElementById('roomCard').onclick = pickRoom;
```

Comment:
Good consistency within your code for all three functions

Reply:
Agree!